docs/concepts/Security-Best-Practices.md

```diff
@@ -117,7 +117,7 @@ For more information about Dependabot alerts & security updates, [see the follow
 
 **📦 Package Consumer**
 
-When using multiple public & private NuGet source feeds, a package can be downloaded from any of the feeds. To ensure your build is predictable and secure from known attacks such as [Dependency Confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610), knowing what specific feed(s) your packages are coming from is a best practice. You can use a single feed or private feed with upstreaming capabilities for protection.
+Use package sources that you trust. When using multiple public & private NuGet source feeds, a package can be downloaded from any of the feeds. To ensure your build is predictable and secure from known attacks such as [Dependency Confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610), knowing what specific feed(s) your packages are coming from is a best practice. You can use a single feed or private feed with upstreaming capabilities for protection.
```

Comment:
at the team meeting we talked about ensuring that we've added this in all places where we have relevant documentation.

Reply:
thanks, I was meaning to comment on the PR, but lost track of time and forgot.  Examples of other places where calling out sources should be trusted include the nuget.config reference docs, and nuget sources add, and dotnet nuget add source.  Maybe also other docs that discuss package sources, like the page on using sources that need authentication.

Reply:
> nuget.config reference docs, and nuget sources add, and dotnet nuget add source

Thank you for the pointers. Pushed new changes. I will create a separate PR for dotnet nuget add/update source.